lira/ASTPrinter.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Text;
+
+namespace Lira;
+
+public class ASTPrinter: IExpr.IVisitor<string>
+{
+
+    public ASTPrinter() {}
```

Comment:
No need to put this now. An empty public constructor is automatically generated by the compiler if none is provided.